program_repair.py

```diff
@@ -22,19 +26,42 @@ def replace(input_string, line, column, length, new_string):
     return result
 
 
+def make_mutants(input_string, line):
+    mutants = []
+    length = len(input_string.splitlines()[line - 1]) + 2
+    for i in range(1, length):
+        mutants.append(insert(input_string, line, i, '\''))
```

Comment:
insert this way is very expensive because split() and join are expensive. Can we rewrite the code so it splits and joins once? So split at the beginning of make_mutants, only pass the one line to insert() and then join all code back in at the end of this?